.github/workflows/nightly-gcp-experiment-pipeline.yml

```diff
@@ -161,28 +172,42 @@ jobs:
         run: |
           gcloud compute instances create litmus-e2e-vm-${{ github.run_number }} \
           --machine-type=f1-micro \
-          --zone=us-central1-a \
+          --zone=us-east1-b \
           --create-disk name=litmus-e2e-first-disk-${{ github.run_number }},size=1GB,device-name=litmus-e2e-first-disk-${{ github.run_number }} \
           --create-disk name=litmus-e2e-second-disk-${{ github.run_number }},size=1GB,device-name=litmus-e2e-second-disk-${{ github.run_number }}
+          gcloud compute disks add-labels litmus-e2e-first-disk-${{ github.run_number }} --labels=disk=litmus-e2e-disk-${{ github.run_number }} --zone=us-east1-b
+          gcloud compute disks add-labels litmus-e2e-second-disk-${{ github.run_number }} --labels=disk=litmus-e2e-disk-${{ github.run_number }} --zone=us-east1-b
 
       - name: Litmus Infra Setup
         if: always()
         run: make build-litmus
+        env:
+          OPERATOR_IMAGE: "${{ github.event.inputs.operatorImage }}"
+          RUNNER_IMAGE: "${{ github.event.inputs.runnerImage }}"
```

Comment:
This won't be required, as it is a nightly build and there are no inputs available.